src/UI/templates/js/MainControls/src/metabar.class.js

```diff
@@ -229,14 +232,24 @@ export default class Metabar {
   /**
    * @return {void}
    */
+  
   disengageAllButtons() {
+    // disengage all buttons in desktop metabar
     this.#jquery(`#${this.#id}.${classForEntries}`)
       .children('li').children(`.btn.${classForBtnEngaged}`)
       .each(
         (i, btn) => {
           disengageButton(this.#jquery(btn));
         },
       );
+    // disengage all buttons in mobile metabar (nested in "more" slate)
+    this.#jquery(`#${this.#id}.${classForEntries}`)
+      .children('li').children(`.${classForSlates}`).children(`.${classForMoreSlate}`).children(`.${classForSingleSlateContent}`).children(`.btn.${classForBtnEngaged}`)
```

Comment:
can we get rid of jquery/consecutive children() calls by using document.querySelectorAll()?
Will also add to readability...
(While you're at it - the part above, too?)